src/apps/vela/arbitrum/vela.vlp-farm.contract-position-fetcher.ts

```diff
@@ -0,0 +1,9 @@
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+
+import { VelaTokenFarmContractPositionFetcher } from '../common/vela.token-farm.contract-position-fetcher';
+
+@PositionTemplate()
+export class ArbitrumVelaVlpFarmContractPositionFetcher extends VelaTokenFarmContractPositionFetcher {
+  velaTokenFarmAddress = '0xfc527781ae973f8131dc26dddb2adb080c1c1f59';
+  pool = { poolId: 0 };
```

Comment:
I'm wondering why it's only building position for first farm instead of the 4 available (pool length on the contract is equal to 4)

Reply:
Attempted to resolve with 740ccb4d977c3e29f5ebc8cb4f4d38279d37f709. This at the same time added support for the VELA farm (example: http://localhost:5001/apps/vela/balances?network=arbitrum&addresses[]=0xc9e1417cb75d8c485d158139bea735d9121d0e7f)... and what ever else there is a farm for. Had to do some changes in `SingleStakingFarmTemplateContractPositionFetcher`, to allow the specific `poolId` to be passed around.